codingQuestion.py

```diff
@@ -1 +1,12 @@
-
+L = int(input())
+N = int(input())
+
+for _ in range(N):
+    W, H = map(int, input().split())
+    

```

Comment:
The comparison operators used in this line should be modified to use '>=' instead of '<'.